docker-compose.yml

```diff
@@ -1,24 +1,54 @@
 version: "3.9"
 
 services:
+    postgres:
+        container_name: postgres
+        image: postgres
+        environment:
+            POSTGRES_USER: root
```

Comment:
Shouldn't user & password be loaded from the `.env` file?

Reply:
Yep, got it. Do you think we should have a specific `.env` variable for each app and a root `.env` for vars shared among many apps variables? It seems like NX recommend it: https://nx.dev/recipes/environment-variables/define-environment-variables

Reply:
Yes, let me create it!